clientapi/routing/membership.go

```diff
@@ -127,18 +127,18 @@ func SendMembership(
 		returnData = struct {
 			RoomID string `json:"room_id"`
 		}{roomID}
+		fallthrough
```

Comment:
Is it ok that `returnData` is empty for invites?

Reply:
Yep, expected response for `/invite` is a 200 and `{}` body.